package.json

```diff
@@ -38,8 +40,8 @@
   },
   "types": "./dist/src/index.d.ts",
   "dependencies": {
-    "@azure-tools/typespec-azure-core": "0.43.0",
-    "@azure-tools/typespec-autorest": "0.43.0",
+    "@azure-tools/typespec-azure-core": ">=0.44.0, <1.0.0",
+    "@azure-tools/typespec-autorest": ">=0.44.0, <1.0.0",
```

Comment:
Would breaks in either of these packages break or affect your tool in any way?

Reply:
Not likely, no. And they always bump the minor version whether it's breaking or not, and I definitely don't want to have to do a token release just to bump the dependency.

There are no actual imports from either package in the code. We simply use autorest to generate the Swagger.